admin/tools/docker/configured/scripts/configure.sh

```diff
@@ -30,14 +30,14 @@ set -e
 usage() {
   cat << EOD
 
-Usage: `basename $0` [options] host
+Usage: `basename $0` [options]
 
   Available options:
     -h          this message
     -m          configure Qserv master, instead of worker by default
 
-  Configure a Qserv worker/master in a docker image. Qserv master fqdn
-  must be provided.
+  Configure a Qserv worker/master in a docker image.
+  Use XROOTD_MANAGER_HOSTNAME to set xrootd configuration.
```

Comment:
I don't see `XROOTD_MANAGER_HOSTNAME` used anywhere else in the code, where is it hidden, is it xrootd variable?